### PR TITLE
test: refactor article summary

### PR DESCRIPTION
**Pull request by craigbilner**
* Split out style and non-style tests
* Mock deps
* Share fixtures between tests and showcase

### CODE REVIEW

File: packages/article-summary/__tests__/web/article-summary-with-style.web.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import shared from "../shared-with-style.web";
+
+describe("ios", () => {
```
Comment (colincclark) [inline packages/article-summary/__tests__/web/article-summary-with-style.web.test.js]: web

File: packages/article-summary/__tests__/web/article-summary.web.test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import shared from "../shared";
+import shared from "../shared.web";
 
-describe("Article Summary tests on Web", () => {
+describe("ios", () => {
```
Comment (colincclark) [inline packages/article-summary/__tests__/web/article-summary.web.test.js]: web....etc

File: packages/article-summary/fixtures/opinion-byline.js [hunk 3]
```diff
@@ -2,43 +2,40 @@ import React from "react";
 import { colours } from "@times-components/styleguide";
 import { ArticleSummaryHeadline, ArticleSummaryContent } from "../";
 
-export default {
+const defaultByline = "Camilla Long, Environment Editor";
```
Comment (colincclark) [inline packages/article-summary/fixtures/opinion-byline.js]: Are we dead-set on using relevant test data in our tests, or can we ensure random data with something like `faker`?

Comment (craigbilner) [reply packages/article-summary/fixtures/opinion-byline.js]: :speaking_head: will pick up with provider fixtures

**Pull request merged**